src/AccessToken.php

```diff
@@ -269,7 +274,7 @@ private function verifyRs256($token, array $certs, $audience = null)
 
         // support HTTP and HTTPS issuers
         // @see https://developers.google.com/identity/sign-in/web/backend-auth
-        $issuers = [self::OAUTH2_ISSUER, self::OAUTH2_ISSUER_HTTPS];
+        $issuers = isset($issuer) ? array($issuer) : [self::OAUTH2_ISSUER, self::OAUTH2_ISSUER_HTTPS];
```

Comment:
Can't do the shorthand ternary here, but could we drop the `isset()` and use shorthand arrays?

```suggestion
        $issuers = $issuer ? [$issuer] : [self::OAUTH2_ISSUER, self::OAUTH2_ISSUER_HTTPS];
```